<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="imageLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTransform_Functions">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Transform Functions</string>
    </property>
    <addaction name="actionRGtoHSV"/>
    <addaction name="actionHSVtoRGB"/>
    <addaction name="actionRGBtoYUV"/>
    <addaction name="actionYUVtoRGB"/>
    <addaction name="actionRGBtoYCbCr"/>
    <addaction name="actionYCbCrtoRGB"/>
    <addaction name="actionRGBtoBGR"/>
    <addaction name="actionBGRtoRGB"/>
    <addaction name="actionRGBtoXYZ"/>
    <addaction name="actionXYZtoRGB"/>
    <addaction name="actionRGBtoYCC"/>
    <addaction name="actionYCCtoRGB"/>
    <addaction name="actionRGBtoHLS"/>
    <addaction name="actionHLStoRGB"/>
    <addaction name="actionRGBtoGray"/>
   </widget>
   <widget class="QMenu" name="menuTransform_Reset">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Transform Reset</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionMedian_Filter"/>
    <addaction name="actionMedian_Filter_5x5"/>
   </widget>
   <widget class="QMenu" name="menuRotate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Rotate</string>
    </property>
    <addaction name="actionRight"/>
    <addaction name="actionLeft"/>
    <addaction name="action180"/>
    <addaction name="actionRotate_on"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTransform_Functions"/>
   <addaction name="menuFilters"/>
   <addaction name="menuRotate"/>
   <addaction name="menuTransform_Reset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRGtoHSV">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGBtoHSV</string>
   </property>
  </action>
  <action name="actionHSVtoRGB">
   <property name="text">
    <string>HSVtoRGB</string>
   </property>
  </action>
  <action name="actionRGBtoYUV">
   <property name="text">
    <string>RGBtoYUV</string>
   </property>
  </action>
  <action name="actionYUVtoRGB">
   <property name="text">
    <string>YUVtoRGB</string>
   </property>
  </action>
  <action name="actionRGBtoYCbCr">
   <property name="text">
    <string>RGBtoYCbCr</string>
   </property>
  </action>
  <action name="actionYCbCrtoRGB">
   <property name="text">
    <string>YCbCrtoRGB</string>
   </property>
  </action>
  <action name="actionRGBtoBGR">
   <property name="text">
    <string>RGBtoBGR </string>
   </property>
  </action>
  <action name="actionBGRtoRGB">
   <property name="text">
    <string>BGRtoRGB</string>
   </property>
  </action>
  <action name="actionRGBtoXYZ">
   <property name="text">
    <string>RGBtoXYZ</string>
   </property>
  </action>
  <action name="actionXYZtoRGB">
   <property name="text">
    <string>XYZtoRGB </string>
   </property>
  </action>
  <action name="actionRGBtoYCC">
   <property name="text">
    <string>RGBtoYCC</string>
   </property>
  </action>
  <action name="actionYCCtoRGB">
   <property name="text">
    <string>YCCtoRGB</string>
   </property>
  </action>
  <action name="actionRGBtoHLS">
   <property name="text">
    <string>RGBtoHLS</string>
   </property>
  </action>
  <action name="actionHLStoRGB">
   <property name="text">
    <string>HLStoRGB</string>
   </property>
  </action>
  <action name="actionRGBtoGray">
   <property name="text">
    <string>RGBtoGray</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionMedian_Filter">
   <property name="text">
    <string>Median Filter 5x1</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="action180">
   <property name="text">
    <string>180</string>
   </property>
  </action>
  <action name="actionMedian_Filter_5x5">
   <property name="text">
    <string>Median Filter 5x5</string>
   </property>
  </action>
  <action name="actionRotate_on">
   <property name="text">
    <string>Rotate on ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
